seshdash/data/db/influx.py

```diff
@@ -269,8 +269,43 @@ def get_latest_measurement_point_site(self, site, measurement_name, database=Non
          return list(self._influx_client.query(query,database=db).get_points())
 
 
+    def get_measurements_latest_point(self, site, measurement_list, database=None):
+        """ Returns a list of elements containing the latest points of provided measurements """
+        
+        # Handling the db to be used
+        db = self.db
+        if database:
+           db = database        
+
+        measurements = ""
+       
+        # Formatting the measurement list
+        for i, measurement in enumerate(measurement_list):
+            measurements += measurement
+            if i == len(measurement_list) - 1:
+                break
+            measurements += ','
+
+        query = "SELECT * FROM %s WHERE site_name='%s' LIMIT 1" % (measurements, site.site_name)
+        points = list(self._influx_client.query(query,database=db).get_points())
+
+        # Creating a dictionary with measurement as key and point as value
+        measurement_dict = {}
+        for i, measurement in enumerate(measurement_list):
+            try:
+                measurement_dict[measurement] = points[i]
+            except:
+                print "No measurements found for %s" % measurement
```

Comment:
let's not use print in the exception but perhaps `logging.warning`? or `logging.error` or `logging.exception`